docs/guides/integrating-optimize.md

```diff
@@ -0,0 +1,122 @@
+---
+id: integrating-optimize
+title: Integrating Optimize
+sidebar_label: Integrating Optimize
+---
+
+## Purpose
+
+The following document provides a basic end-to-end glance into Optimize and its features for new business users.
+
+Optimize offers business intelligence tooling for Camunda enterprise customers. By leveraging data collected during process execution, you can access reports, share process intelligence, analyze bottlenecks, and examine areas in business processes for improvement.
+
+With Optimize, you can review heatmap displays for a closer look at the number of instances that took longer than average, based on duration distribution. As a result, reap actionable insights and rapidly identify the constraints of your system.
+
+See [What is Optimize](https://camunda.com/products/camunda-platform/optimize/reports/) for an in-depth overview of Optimize’s capabilities.
+
+[//]:# (The above is just a placeholder link. Amara, should we add a landing page for Optimize? This was the text I moved below this original Optimize guide.)
```

Comment:
Let's add the landing page with the other Optimize migration work, knowing this will likely get merged to master ahead of time. Keep this placeholder for now and we can update it in the Optimize migration PR.